src/codegen/cu_ast.rs

```diff
@@ -163,6 +165,7 @@ pub(super) enum GpuAddrSpace {
 #[derive(Clone, Debug)]
 pub(super) enum Ty {
     Scalar(ScalarTy),
+    Atomic(ScalarTy),
```

Comment:
Will this be used to generate `std::cuda::atomic`?

Reply:
yes, is fixed